Libraries/LibJS/Value.h

```diff
@@ -36,6 +36,7 @@ namespace JS {
 class Value {
 public:
     enum class Type {
+        NotDefined,
```

Comment:
This class is meant to represent a JavaScript value, and there is no such thing as a "not defined" value in JS.
If you want to communicate the absence of a value somewhere, please use a wrapper like `Optional<Value>`